js/screens/Profile/ProfileContainer.js

```diff
@@ -18,5 +18,7 @@ class ProfileContainer extends Component {
     return <Profile user={this.state.user} />;
   }
 }
-
+ProfileContainer.propTypes = {
```

Comment:
You are not receiving any props for ProfileContainer so you don't need to check for propTypes. Also, don't import it.